core/utils/image_urls.py

```diff
@@ -1,16 +1,30 @@
 from django.conf import settings
-
 from django.utils.http import urlquote
 
+RESIZE = 0
```

Comment:
Nah, don't worry about a comment!  It's clear what it is happening from this snippet of code, I was just kind of thinking that in the future it might not be obvious why RESIZE is returning 0 or something like that.